### PR TITLE
Create sleekflow.md

### PR DESCRIPTION
**Pull request by Brandoncharleskhoo1**
# Amplitude Developer Docs PR

## Description

Describe your changes. 

## Deadline

When do these changes need to be live on the site?

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [ ] Doc update.
- [ ] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [ ] My documentation follows the style guidelines of this project.
- [ ] I previewed my documentation on a local server using `mkdocs serve`.
- [ ] Running `mkdocs serve` didn't generate any failures.
- [ ] I have performed a self-review of my own documentation.

@amplitude-dev-docs

### CODE REVIEW

File: docs/data/sources/sleekflow.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+- You need a Sleekflow account and an Amplitude plan.
+- This integration supports these identifiers:
+    - User ID: Email address
+    - Connection type: Server
+
+## Setup
+
+1. In Sleekflow, click on the **Automation** tab.
+2. Click on **Create new rule**.
```
Comment (markzegarelli) [inline docs/data/sources/sleekflow.md:26]:
```suggestion
2. Click **Create new rule**.
```

File: docs/data/sources/sleekflow.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+3. Click **Customized a rule**.
```
Comment (markzegarelli) [inline docs/data/sources/sleekflow.md:27]: Customize a rule? Customized rule?

File: docs/data/sources/sleekflow.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+## Considerations
+
+- You need a Sleekflow account and an Amplitude plan.
+- This integration supports these identifiers:
+    - User ID: Email address
+    - Connection type: Server
+
+## Setup
+
+1. In Sleekflow, click on the **Automation** tab.
```
Comment (markzegarelli) [inline docs/data/sources/sleekflow.md:25]:
```suggestion
1. In Sleekflow, navigate to the **Automation** tab.
```

File: docs/data/sources/sleekflow.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+2. Click on **Create new rule**.
+3. Click **Customized a rule**.
+4. Click on **Incoming messages**.
```
Comment (markzegarelli) [inline docs/data/sources/sleekflow.md:28]:
```suggestion
4. Click **Incoming messages**.
```

File: docs/data/sources/sleekflow.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+5. Under **Conditions**, click on **Add condition**, and select **Phone Number**.
+6. Under **contains**, select "**is known as**".
+7. Under **Actions**, click on the big X near the right.
+8. Under **Add action**, select "Send webhook".
+9. Click **Publish rule**.
+10. In the blank, fill in: https://sleekflow-amplitude-f375paklaq-uc.a.run.app/receive-message?apiKey={api_key}.
```
Comment (markzegarelli) [inline docs/data/sources/sleekflow.md:34]: Where does a user find this API key? Amplitude? Sleekflow?

File: docs/data/sources/sleekflow.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+11. Repeat steps 2-8, whereas in step 4, instead of selecting **Incoming Messages**, select **Outgoing Messages**.
+12. In the same blank as step 9, fill in: https://sleekflow-amplitude-f375paklaq-uc.a.run.app/send-message?apiKey={api_key}.
+13. Click **Publish rule**.
+
+## Use cases
+
+1. **User Behavior Analysis:** You can track how users engage with your brand or business through different messaging platforms. This includes monitoring when users open messages, click on links, make inquiries, or complete transactions. This data helps you understand user behavior and preferences.
```
Comment (markzegarelli) [inline docs/data/sources/sleekflow.md:41]:
```suggestion
1. **User Behavior Analysis:** Track how users engage with your brand or business through different messaging platforms. Monitor when users open messages, click links, make inquiries, or complete transactions. This data helps you understand user behavior and preferences.
```

File: docs/data/sources/sleekflow.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+2. **Conversion Tracking:** By sending events related to conversions (e.g., completing a purchase, signing up for a newsletter, or requesting more information) to Amplitude, you can measure the effectiveness of your messaging campaigns on different platforms. This information can help you optimize your marketing strategies.
```
Comment (markzegarelli) [inline docs/data/sources/sleekflow.md:42]:
```suggestion
2. **Conversion Tracking:** When you send events related to conversions (like completing a purchase, signing up for a newsletter, or requesting more information) to Amplitude, you can measure the effectiveness of your messaging campaigns on different platforms. This information can help you optimize your marketing strategies.
```

File: docs/data/sources/sleekflow.md [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+3. **User Segmentation:** Amplitude allows you to create user segments based on specific actions and behaviors. You can use Sleekflow's event data to create custom segments of users who have interacted with your business in particular ways on different channels. This segmentation can be used for targeted messaging and marketing efforts.
```
Comment (markzegarelli) [inline docs/data/sources/sleekflow.md:43]:
```suggestion
3. **User Segmentation:** Amplitude allows you to create user segments based on specific actions and behaviors. Use Sleekflow's event data to create custom segments of users who've interacted with your business in particular ways on different channels. Use this segmentation for targeted messaging and marketing efforts.
```

**Review by markzegarelli — CHANGES_REQUESTED** [review]
@Brandoncharleskhoo1 Can you share a screenshot of the UI from Sleekflow? I think there is a better way to document it, but need to see what it looks like first.

File: docs/data/sources/sleekflow.md [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+3. Click **Customized a rule**.
+4. Click on **Incoming messages**.
+5. Under **Conditions**, click on **Add condition**, and select **Phone Number**.
+6. Under **contains**, select "**is known as**".
+7. Under **Actions**, click on the big X near the right.
+8. Under **Add action**, select "Send webhook".
+9. Click **Publish rule**.
+10. In the blank, fill in: https://sleekflow-amplitude-f375paklaq-uc.a.run.app/receive-message?apiKey={api_key}.
+11. Repeat steps 2-8, whereas in step 4, instead of selecting **Incoming Messages**, select **Outgoing Messages**.
+12. In the same blank as step 9, fill in: https://sleekflow-amplitude-f375paklaq-uc.a.run.app/send-message?apiKey={api_key}.
```
[the remark this answers is not in the document]
Comment (markzegarelli) [reply docs/data/sources/sleekflow.md:36]: @Brandoncharleskhoo1 Can you clarify this one?

File: docs/data/sources/sleekflow.md (hunk 5 above)
Comment (markzegarelli) [reply docs/data/sources/sleekflow.md:34]: @Brandoncharleskhoo1 Can you clarify this one?

File: docs/data/sources/sleekflow.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (Brandoncharleskhoo1) [reply docs/data/sources/sleekflow.md:36]: @jamesbrandon @isaacli430 Are you able to clarify the steps here?

File: docs/data/sources/sleekflow.md (hunk 2 above)
Comment (Brandoncharleskhoo1) [reply docs/data/sources/sleekflow.md:27]: @jamesbrandon @isaacli430 Are you able to clarify the steps here as well?

File: docs/data/sources/sleekflow.md (hunk 5 above)
Comment (Brandoncharleskhoo1) [reply docs/data/sources/sleekflow.md:34]: @jamesbrandon @isaacli430 Are you able to clarify the steps here as well?

**Pull request closed without merging**